client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -348,6 +349,12 @@ export const openPrintingFlow = ( orderId, siteId ) => ( dispatch, getState ) =>
 	waitForAllPromises( promisesQueue ).then( () =>
 		tryGetLabelRates( orderId, siteId, dispatch, getState )
 	).then( () => {
+		//compatibility - only add the email_receipt if the plugin and the server support it
```

Comment:
What is this comment referring to?

Reply:
Sorry, I think it was leftover on some changes. I'll remove this one. Thanks for spotting this.